t/syntax.t

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+
+use Test::More;
+use Test::Strict;
+use Test2::AsyncSubtest;
+
+use FindBin qw/$Bin/;
+use POSIX qw( waitpid WNOHANG );
+use Time::HiRes qw( sleep );
+
+local $Test::Strict::TEST_WARNINGS = 1;
+
+my @dirs  = ('t', 'bin', "$Bin/../share/shutter/resources/modules/");
+my @files = Test::Strict::_all_perl_files(@dirs);
```

Comment:
This is a private function of Test::Strict? It's not documented at https://metacpan.org/pod/Test::Strict

Why `all_perl_files_ok` was not enough?

Reply:
10 seconds is good enough. Additional 6 seconds don't deserve the added complexity